components/inlinealert/index.css

```diff
@@ -12,10 +12,20 @@ governing permissions and limitations under the License.
 
 .spectrum-InLineAlert {
   /* Font */
-  --spectrum-inlinealert-font-weight: var(--spectrum-bold-font-weight);
+  --spectrum-inlinealert-heading-font-family: var(--spectrum-heading-sans-serif-font-family);
```

Comment:
This is something that I just ran into with the card component that we need to ask Garth about. The `--spectrum-heading-sans-serif-font-family` and `--spectrum-body-sans-serif-font-family` tokens are incorrect which you can see in the screenshot that the font it's showing is a default serif font, not sans-serif. I think the issue is that the token is set to `Adobe Clean` and it needs to be set to `abode-clean`
![Screenshot 2023-03-29 at 1 49 15 PM](https://user-images.githubusercontent.com/99203545/228624986-61a5e774-37b0-4475-b574-b455df60311b.png)

Reply:
I've noticed this as well while working on other components. The Figma defined font-family Figma tokens do not translate to a usable font stack. I have been using `--spectrum-sans-font-family-stack`, and have been meaning to ask if this is correct.

Reply:
Oh interesting: this actually looks fine to me, but I also have Adobe Clean installed locally. Theoretically, it should be falling back to `adobe-clean` which would be provided by Typekit on the docs site. Might need to dig into this a little more.

Reply:
I wrote a bit more about this in [Slack](https://adobedesign.slack.com/archives/G019JTYMT6H/p1680114911681059), but: [we have font stacks](https://github.com/adobe/spectrum-css/blob/32efe27a3dbc89da7bb9882004eebe318e0ce060/components/tokens/custom-spectrum/custom-vars.css#L38-L47) that we'll need to use for `font-family` things. Sorry that we didn't discuss this earlier. Not all of our components define `font-family` and this is the first time I think it's come up with y'all.